docs/content/getting-started/_index.md

```diff
@@ -75,7 +75,7 @@ The beauty of web components is that they have much of the styling built-into th
 ### Container components (see also container notes below)
 
 ```
-<body>
+<body unresolved>
```

Comment:
I would prefer that the body tag be removed from this example.  I don't think it adds anything, since the object of the example is pfe-card.  Unless I'm missing something about why `<body>` is relevant in this example, I think this looks better:

```html
    <pfe-card pfe-color="darkest">
        <p>Hello world.</p>
    </pfe-card>
```

Reply:
also agreed; pushed in 05f2510de4c46a5587cf9f695481e22269f966da